styles.css

```diff
@@ -1,6 +1,108 @@
-body{
-    color: #bae1ff;
+:root {
+    --primary-color: #370140;
+    --secondary-color: #bae1ff;
 }
-h1{
-    color:#6b37c1;
+
+/* Create and style grid container for the banner */
+
+.center {
```

Comment:
Great use in margin left and right, its a clearer direction in having the content accurately in centre ✨